zkdoc/release-note

```diff
@@ -80,6 +80,8 @@ ZK 10.0.0
         z-combobox-emptySearchMessage -> z-combobox-empty-search-message
         z-combobox-emptySearchMessage-hidden -> z-combobox-empty-search-message-hidden
   + Disable UUID recycle mechanism by default.
+  + Customize the tooltip on the icon of LabelImagElelement (see ZK-5502).
+  + Customize multiple icons and tooltips on the same LabelImagElelement (see ZK-5503).
```

Comment:
typo.
and with these two lines, I am not able to realize what should I notice in ZK 10